rustls-libssl/src/entry.rs

```diff
@@ -597,6 +597,17 @@ entry! {
     }
 }
 
+entry! {
+    pub fn _SSL_CTX_set_cipher_list(_ctx: *mut SSL_CTX, s: *const c_char) -> c_int {
+        match try_str!(s) {
+            "HIGH:!aNULL:!MD5" => C_INT_SUCCESS,
```

Comment:
So boring :laughing: No Rustls support for STREEBOG256!?! Or IDEA?!? :zany_face: